### DISCUSSION TITLE
Contact Information to Casadi/Industry Coop?

### DISCUSSION THREAD

**Discussion by pkei1234** [casadi/casadi#3417] [Q&A]
Hi, I read in the Casadi Documentation that I can find help and contact <someone> regarding industrial cooperation. I'm not 100% sure if this is the kind of help I need but I would like to contact someone who knows (alot more than me) about capabilities of casadi within commercial applications (and legalities of it really).

However, i do not see contact information, nor do I know the name of anyone who I can talk to about this? Who is responsible for this kind of query, and I would very much like to talk to someone about getting help. Thanks.

Kindest Regards,
Dr. Peter Kei

```
1.2. Help and support[¶](https://web.casadi.org/docs/#help-and-support)
If you find simple bugs or lack some feature that you think would be relatively easy for us to add, the simplest thing is simply to write to the forum, located at http://forum.casadi.org/. We check the forum regularly and try to respond as quickly as possible. The only thing we expect for this kind of support is that you cite us, cf. [Section 1.3](https://web.casadi.org/docs/#sec-citing), whenever you use CasADi in scientific work.

If you want more help, we are always open for academic or industrial cooperation. An academic cooperation usually take the form of a co-authorship of a peer reviewed paper, and an industrial cooperation involves a negotiated consulting contract. Please contact us directly if you are interested in this.

```

**Comment by jgillis** [comment]
Please refer to https://web.casadi.org/support/
